nest/src/main.rs

```diff
@@ -129,19 +159,27 @@ fn main() {
         )
         .get_matches();
```

Comment:
About the whole above, it's all written in the style "sets", "does" etc. usually (in mans I read anyway) it reads more like "print C-style escapes for nongraphic characters".

Reply:
Honestly, i know and kind-of did it on purpose because it feels more natural to me. But I think you're right and we should follow conventions here.

This raises an other question: what about documentation? Should we write `Returns the sum of the two argument` or `Return the sum of two argument`?
@Pedrochu might have to re-write the whole doc if we change our mind.

Reply:
I think we can leave the current style, it's just that I was a little suprised.